src/main/java/pro/sky/telegrambot/listener/Scheduled.java

```diff
@@ -0,0 +1,29 @@
+package pro.sky.telegrambot.listener;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pro.sky.telegrambot.model.NotificationTask;
+import pro.sky.telegrambot.repository.NotificationTaskRepository;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class Scheduled {
+
+    @Autowired
+    private final NotificationTaskRepository notificationTaskRepository;
+
+    public Scheduled(NotificationTaskRepository notificationTaskRepository) {
+        this.notificationTaskRepository = notificationTaskRepository;
+    }
+
+    public List<NotificationTask> runTask() {
+        return notificationTaskRepository.findAll().stream()
+                .filter(f -> f.getLocalDateTime().equals(LocalDateTime.now().truncatedTo(ChronoUnit.MINUTES)))
+                .collect(Collectors.toList());
```

Comment:
Ты вот добавил в репозиторий метод findAllByExecDateLessThan.
Теперь его тут нужно использовать:
```suggestion
        return notificationTaskRepository.findAllByExecDateLessThan(LocalDateTime.now());
```